tests/features/env-info.feature

```diff
@@ -21,11 +21,34 @@ Feature: Displaying environmental information
       PHP Version
     """
 
+  @vcr env-info.yml
```

Comment:
@TeslaDethray Here is a test for the new TERMINUS_SITE / TERMINUS_ENV feature. Did you want any additional coverage?

Reply:
Yes. PHP Unit tests for the new functions.